docs/locale/ja/LC_MESSAGES/bugDescriptions.po

```diff
@@ -12543,3 +12543,18 @@ msgstr ""
 #~ "</p>"
 #~ msgstr ""
 
+#~ msgid ""
```

Comment:
Why add these lines?

Reply:
I don't know. I just modified `spotbugs/etc/messages.xml`  and executed `./build.sh` in the **doc** directory.

Reply:
we can safely remove these lines.